enip_backend/export/schemas/national.example.json

```diff
@@ -0,0 +1,443 @@
+{
+  "national_summary": {
+    "P": [
+      {
+        "name": "Clinton",
+        "party": "Dem",
+        "winner": false,
+        "popVote": 65853516,
+        "popPct": 0.49,
+        "statesWon": 21,
+        "electWon": 232
+      },
+      {
+        "name": "Trump",
+        "party": "GOP",
+        "winner": true,
+        "popVote": 62984825,
+        "popPct": 0.46,
+        "statesWon": 30,
+        "electWon": 306
+      },
+      {
+        "name": "Johnson",
+        "party": "Lib",
+        "winner": false,
+        "popVote": 4489221,
+        "popPct": 0.03,
+        "statesWon": 0,
+        "electWon": 0
+      }
+    ],
+    "S": [
+      {
+        "won": 22,
+        "party": "Dem"
+      },
+      {
+        "won": 11,
+        "party": "GOP"
+      },
+      {
+        "won": 2,
```

Comment:
on the back-end I think we'll want to add this in to the Dem total

Reply:
Not going to worry about this because the holdovers are hardcoded

Reply:
this is a moot point now that Bernie and Angus are part of holdovers, and we'll treat Al Gross as Dem